js/generate-index.js

```diff
@@ -130,6 +130,7 @@ loadLocalizedUrl('../js/template-index.html', lang)
       let navLi = document.createElement('li');
       navLi.innerHTML = templateTocItem;
       navLi.querySelector('a').href = page.url;
+      navLi.querySelector('img').src = page.icon;
```

Comment:
Is it better to use `.icon img` as the selector? I'm not quite sure because the template HTML does not contain too many elements...

Reply:
I'm fine either way, so updated to `.icon img`. Who knows, we may complete the HTML template with other images later on, in which case the more specific selector will prove useful!